app/views/notes/explore.js.erb

```diff
@@ -0,0 +1,8 @@
+$('#notes-div').append('<%= j render @notes %>');
+<% if @notes.next_page %>
+  $('.pagination').replaceWith('<%= j will_paginate @notes %>');
+<% else %>
+  $(window).off('scroll');
+  $('.pagination').remove();
+<% end %>
+$('#loader').addClass('hidden');
```

Comment:
sir @AndreMigz . To remove this "minus" sign. add empty newline at the bottom. Github is following POSIX standard.

Reply:
HI @iprog21 I already committed the suggested changes :rocket: